dimod/core/bqm.py

```diff
@@ -141,6 +143,22 @@ def __setitem__(self, v, bias):
         # inherits its ability to reshape the bqm from the `.set_linear` method
         self._bqm.set_linear(v, bias)
 
+    def min(self):
+        try:
+            return cylmin(self._bqm)
+        except TypeError:
+            pass
```

Comment:
Why would this fail?

Reply:
`cylin` and others are defined only for the cyBQMs. So this will fail for AdjDict for instance.